src/python/pants_release/start_release.py

```diff
@@ -203,18 +227,61 @@ def splice_into_file(release_info: ReleaseInfo, formatted: Formatted) -> None:
     file_name.write_text(splice(existing_contents, formatted.external))
 
 
-def main() -> None:
-    args = create_parser().parse_args()
-    release_info = ReleaseInfo.determine(args.new)
+def update_changelog(release_info: ReleaseInfo) -> Formatted:
     branch_sha = git_fetch(release_info.branch)
     date = datetime.date.today()
-    entries = [prepare_sha(sha) for sha in relevant_shas(args.prior, branch_sha)]
+    entries = [prepare_sha(sha) for sha in relevant_shas(branch_sha)]
 
     formatted = format_notes(release_info, entries, date)
     splice_into_file(release_info, formatted)
-    print(f"\nCommit {release_info.notes_file_name()} and create a PR.\n\n{formatted.internal}")
+
+    return formatted
+
+
+def update_contributors(release_info: ReleaseInfo) -> None:
+    if release_info.branch == "main":
+        CONTRIBUTORS_PATH.write_text(
+            "Created as part of the release process.\n\n+ "
+            + "\n+ ".join(sorted_contributors(git_range="HEAD"))
+            + "\n"
+        )
+
+
+def update_version(release_info: ReleaseInfo) -> None:
+    if release_info.branch == "main":
+        VERSION_PATH.write_text(f"{release_info.version}\n")
+
+
+def commit_and_pr(release_info: ReleaseInfo, formatted: Formatted, release_manager: str) -> None:
+    title = f"Prepare {release_info.version}"
+    branch = f"automation/release/{release_info.version}"
+
+    git("checkout", "-b", branch)
+    git("add", str(VERSION_PATH), str(CONTRIBUTORS_PATH), str(release_info.notes_file_name()))
+    git("commit", "-m", title)
+    git("push", "origin", "HEAD")
+
+    github_pr_create(
+        base="main",
+        head=branch,
+        title=title,
+        body="\n\n".join([BASE_PR_BODY, formatted.internal]),
+        assignee=release_manager,
+        labels=["automation:release-prep"],
```

Comment:
Note branch name and this label name, per the discussion of conventions in the PR description.